pkg/compliance/checks/builder.go

```diff
@@ -161,6 +187,11 @@ func (b *builder) Close() error {
 }
 
 func (b *builder) ChecksFromRule(meta *compliance.SuiteMeta, rule *compliance.Rule) ([]check.Check, error) {
+	if b.scopeFilter != nil && !b.scopeFilter(rule.Scope) {
```

Comment:
I refactored this quite a bit in the original PR to allow various options for loading rules/checks, let me know if you need any help with rebase..